content/rs/installing-upgrading/_index.md

```diff
@@ -1,12 +1,188 @@
+Repeat this process for each node in the cluster and then:
+
+1. [Create]({{< relref "/rs/administering/cluster-operations/new-cluster-setup.md" >}})
+or [join]({{< relref "/rs/administering/cluster-operations/adding-node.md" >}}) an RS cluster.
+1. [Create a database]({{< relref "/rs/administering/database-operations/creating-database.md" >}}).
+
+    For geo-distributed Active-Active replication, create an [Active-Active]({{< relref "/rs/administering/database-operations/create-crdb.md" >}}) database.
+
+### Installation questions
+
+During the installation process, you must answer a few questions to configure the node for your environment.
+These installation questions are:
+
+- **Linux swap file** - `Swap is enabled. Do you want to proceed? [Y/N]?`
+
+    We recommend that you [disable Linux swap]({{< relref "/rs/installing-upgrading/configuring/linux-swap.md" >}}) in the operating system configuration
+    to give RS control of the memory allocation.
+
+- **Automatic OS tuning** - `Do you want to automatically tune the system for best performance [Y/N]?`
+
+    To let the RS installation optimize the OS for Redis Enterprise, answer `Y`.
+    The installation process prompts you for additional information.
+
+    The `/opt/redislabs/sbin/systune.sh` file contains details about the tuning process.
+
+- **Network time** - `Do you want to set up NTP time synchronization now [Y/N]?`
+
+    Redis Enterprise requires that all nodes of the cluster have synchronized time.
+    You can either let the installation process configure NTP
+    or you can [configure NTP manually]({{< relref "/rs/administering/designing-production/synchronizing-clocks.md" >}}).
+
+- **Firewall ports** - `Would you like to open RedisLabs cluster ports on the default firewall zone [Y/N]?`
+
+    RS requires the node to have [specific network ports]({{< relref "/rs/administering/designing-production/networking/port-configurations.md" >}}) open.
+    You can either:
+
+    - Answer `Y` to let the installation process open these ports.
+    - Answer `N` and configure the firewall manually for [RHEL/CentOS firewall]({{< relref "/rs/installing-upgrading/configuring/centos-rhel-7-firewall.md" >}}).
+    - Answer `N` and configure the firewall on the node manually for your OS.
+
+- **Installation verification (rlcheck)** - `Would you like to run rlcheck to verify proper configuration? [Y/N]?`
+
+    We recommend that you run the rlckeck installation verification to make sure that the installation completed succesfully.
+    If you want to run this verification at a later time, you can run: `/opt/redislabs/bin/rlcheck`
+
+### Installation answer file
+
+To avoid answering the installation questions during the installation process,
+you can prepare an answer file and use it to do a silent installation.
+
+To install RS with an answer file:
+
+1. Prepare the answer file with the answers to the questions.
+
+    For example:
+
+    ```sh
+    ignore_swap=no
```

Comment:
Is this the complete list of parameters in the answer file? If so, I think we should make that clear somehow. Maybe before the example, we say:

The answer file should contain the following five parameters, each followed by a `=` and then either `yes` or `no` to whether the given option should be enabled.